example/index.ts

```diff
@@ -30,7 +24,11 @@ const getExecutionCommand = (program: string): string => {
                 throw new Error('Environment variable "ProgramFiles(x86)" is not set on Windows');
             }
             const jarPath = path.join(programFilesX86, 'SIL', appName, 'bin', jarName);
-            return `java -jar "${jarPath}"`;
+            let javaPath = 'java';
```

Comment:
This only deals with the case with java not being on the path for Windows.

Reply:
I can move it out so it works anywhere.